app/device/altimeter.py

```diff
@@ -0,0 +1,76 @@
+from smbus2 import SMBus
+from math  import log1p
+
+#special char
+deg = u'\N{DEGREE SIGN}'
+
+class Altimeter (object):
+
+    MPL3115A2_ADDRESS = (0x60)
+
+    #REGISTERS
+
+    STATUS = (0x00)
+    OUT_P_MSB = (0x01)
+    OUT_P_DELTA_CSB = (0x08)
+    OUT_P_DELTA_MSB = (0X07)
+    OUT_T_MSB = (0x04)
+    WHOAMI = (0x0C)
+    CTRL_REG1 = (0x26)
+    PT_DATA_CFG = (0x13)
+    bus = SMBus(1)
+
+    def __init__(self):
+
+
+        #Check if device is connected
+        #whois = bus.read_byte_data(self.MPL3115A2_ADDRESS, self.WHOAMI)
+        #if(id!=0xc4):
+        #    print("Device not connected")
+        #    exit(1)
+
+        #Set oversample rate to 128
+        current_setting = self.bus.read_byte_data(self.MPL3115A2_ADDRESS, self.CTRL_REG1)
+        new_setting = 0xb8
+        self.bus.write_byte_data(self.MPL3115A2_ADDRESS, self.CTRL_REG1, new_setting)
+
+        # Enable event flags
+        self.bus.write_byte_data(self.MPL3115A2_ADDRESS, self.PT_DATA_CFG, self.OUT_P_DELTA_MSB)
+
+        # Toggel One Shot
+        setting = self.bus.read_byte_data(self.MPL3115A2_ADDRESS, self.CTRL_REG1)
+        if (setting & 0x02) == 0:
+            self.bus.write_byte_data(self.MPL3115A2_ADDRESS, self.CTRL_REG1, (setting | 0x02))
+
+    def temp(self):
+        #print("Waiting for data")
+        status = self.bus.read_byte_data(self.MPL3115A2_ADDRESS, self.STATUS)
+        while(status & self.OUT_P_DELTA_CSB)==0:
+            #print bin(status)
+            status = self.bus.read_byte_data(self.MPL3115A2_ADDRESS, self.STATUS)
+            time.sleep(0.5)
+
+        #Reading sensor data
+        t_data = self.bus.read_i2c_block_data(self.MPL3115A2_ADDRESS, self.OUT_T_MSB, 2)
+
+        t_msb = t_data[0]
+        t_lsb = t_data[1]
+
+        #Temperature in Kelvin
+        temp = t_msb + (t_lsb >> 4)/16.0 + 273.15
+
+        return temp
+
+
+    def altitude(self):
+        a_data = self.bus.read_i2c_block_data(self.MPL3115A2_ADDRESS, self.OUT_P_MSB, 3)
+
+        alt_int_bin = a_data[0] + a_data[1] + a_data[2][0] + a_data[2][1]
```

Comment:
this should not work? you should have to do the following since these are bytes:
```
a_data[0] << 18 | a_data[1] << 10 | a_data[2][0] << 1 | a_data[2][1]
```
using the bit shifting operations